src/tools/fuzzing.h

```diff
@@ -2180,21 +2190,21 @@ class TranslateToFuzzReader {
                             make(v128),
                             make(v128)});
       }
-      case anyref: // there's no binary ops for anyref
-      case exnref: // there's no binary ops for exnref
-      case none:
-      case unreachable:
+      default:
         WASM_UNREACHABLE("unexpected type");
     }
     WASM_UNREACHABLE("invalid type");
   }
 
-  Expression* buildSelect(const ThreeArgs& args) {
-    return builder.makeSelect(args.a, args.b, args.c);
+  Expression* buildSelect(const ThreeArgs& args, Type type) {
+    return builder.makeSelect(args.a, args.b, args.c, type);
   }
 
   Expression* makeSelect(Type type) {
-    return makeDeNanOp(buildSelect({make(i32), make(type), make(type)}));
+    // try to have a subtype for one arm
```

Comment:
I can make the other arm to be a subtype now with no problem :) I just randomly made only one arm a subtype, but maybe we don't need to do that after all because subtype includes the type itself as well.